### PR TITLE
Allow admins to get join us requests

### PR DESCRIPTION
**Pull request by hazemessam**
## Issue that this pull request solves

 Closes: #680

## Proposed changes

- Added GET API to allow admins get join us requests.
- Test the API endpoint.

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

### CODE REVIEW

File: backend/app/routes/joinUs/index.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
 const router = require('express').Router({ mergeParams: true });
 const postJoinUs = require('./post');
+const getJoinUs = require('./get');
 const JoinUsValidationSchema = require('./@validationSchema');
 const validation = require('../../../helpers/middlewares/validation');
+const {authMiddleware} = require('../../../helpers/middlewares/auth')
 
-router.post('/joinUs', validation(JoinUsValidationSchema), postJoinUs);
+router.post('/', validation(JoinUsValidationSchema), postJoinUs);
+// router.get('/', authMiddleware, getJoinUs);
```
Comment (Kajol-Kumari) [inline backend/app/routes/joinUs/index.js:9]: Why did we removed the authmiddleware?

File: backend/app/routes/index.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const broadcast = require('./broadcast');
 const faq = require('./faq/post');
 const getFaq = require('./faq/getFaq');
 const deleteFaq = require('./faq/deleteFaq');
-const joinUs = require('./joinUs/post');
```
Comment (Kajol-Kumari) [inline backend/app/routes/index.js:10]: This was the endpoint for adding the data to db. 
[Question] Why did we remove it?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@hazemessam Please address the below comments. Rest Looks good to me :)

File: backend/app/routes/joinUs/index.js (hunk 1 above)
Comment (hazemessam) [reply backend/app/routes/joinUs/index.js:9]: Sorry, I commented it while I'm debugging.

File: backend/app/routes/index.js (hunk 2 above)
Comment (Kajol-Kumari) [reply backend/app/routes/index.js:10]: This question remains unanswered. Please don't remove any existing route

**Comment by Kajol-Kumari** [comment]
@hazemessam Please write the test of the api as well

**Review by Kajol-Kumari — APPROVED** [review]
Nice and Clean!

LGTM 👍

**Pull request merged**